scalding-serialization-macros/src/main/scala/com/twitter/scalding/serialization/macros/impl/ordered_serialization/TreeOrderedBuf.scala

```diff
@@ -102,19 +102,25 @@ object TreeOrderedBuf {
         q"""
 
       override def dynamicSize($element: $typeName): Option[Int] = {
-        val $tempLen = payloadLength($element) match {
-          case _root_.com.twitter.scalding.serialization.macros.impl.ordered_serialization.runtime_helpers.NoLengthCalculation => None
-          case _root_.com.twitter.scalding.serialization.macros.impl.ordered_serialization.runtime_helpers.ConstLen(l) => Some(l)
-          case _root_.com.twitter.scalding.serialization.macros.impl.ordered_serialization.runtime_helpers.DynamicLen(l) => Some(l)
-        }
-        (if ($tempLen.isDefined) {
-          // Avoid a closure here while we are geeking out
-          val innerLen = $tempLen.get
-          val $lensLen = posVarIntSize(innerLen)
-          Some(innerLen + $lensLen)
-       } else None): Option[Int]
+        writtenRecords += 1L
```

Comment:
its really a poorly named variable, will modify. -- Its counting points at which we would measure the length of our payload. So either by calling dynamic size as an accessor or via the write method.